examples/recursive-src/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "recursive-src",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "rebel -show-actions-run-verbose -verbose -show-error-dependency-paths -brief-error-summary -show-buildable-discovery"
```

Comment:
I have it. But writing all that in npm scripts looked verbose so choose few flags and the output is kind of good enough for the debugging I do.